packages/esm-patient-orders-app/src/components/order-price-details.test.tsx

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+import { screen } from '@testing-library/react';
+import OrderPriceDetailsComponent from './order-price-details.component';
+import { useOrderPrice } from '../hooks/useOrderPrice';
+import { renderWithSwr } from 'tools';
+import { useTranslation } from 'react-i18next';
+import { mockOrderPriceData } from '../../../../__mocks__/order-price-data.mock';
```

Comment:
```suggestion
import { mockOrderPriceData } from '__mocks__';
```

See the note about test mocks [here](https://o3-docs.openmrs.org/docs/coding-conventions/project-structure.en-US) for why this works.